Tasks/JenkinsDownloadArtifacts/task.json

```diff
@@ -293,7 +293,7 @@
         "UploadingAttachment": "Uploading %s as attachment",
         "SuccessfullyUploadedCommitsAttachment": "Successfully uploaded commits attachment",
         "GetCommitMessages": "Getting commit messages from commits",
-        "GetCommitMessagesFailed": "Getting commits message failed with an error %s",
+        "GetCommitMessagesFailed": "Getting commits message failed with an error %s. \n Parsed Json: %s",
```

Comment:
This is the unparsed (failure happens when parsing), right? Reword to Commits: %s?